src/ansible_navigator/actions/run.py

```diff
@@ -279,6 +282,14 @@ def run(self, interaction: Interaction, app: AppPublic) -> Union[Interaction, No
 
         self.steps.append(self._plays)
 
+        # Show a notification until the first the first message from the queue is processed
+        if self._subaction_type == "run":
+            messages = ["Preparing for automation, please hold on..."]
```

Comment:
Nit: probably better to keep idioms out of user-facing strings, especially since we don't currently do any translations.

```suggestion
            messages = ["Preparing for automation, please wait..."]
```